monai/bundle/utils.py

```diff
@@ -155,6 +155,15 @@
     },
 }
 
+DEFAULT_CLEARML_STATS_SETTINGS = {
+
+}
+
+DEFAULT_CLEARML_IMAGE_SETTINGS = {
+
+}
+
+
```

Comment:
I would suggest to remove these empty dicts, let's add them when really want to implement the requirements.

Thanks.